src/styles/header.scss

```diff
@@ -0,0 +1,85 @@
+// #region header
+.header {
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.navi-bar {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.navi-menu {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.navi-item {
+  margin: 0;
+  padding-left: 20px;
```

Comment:
It's a good practice to use `margin` for outer spacing (space between elements) and `padding` for inner spacing (space within an element). In your case, you are using `padding-left` to create space between `.navi-item` elements. This could be better achieved using `margin-right` on `.navi-item`.